pennylane_lightning/core/simulators/lightning_gpu/bindings/LGPUBindings_nb.hpp

```diff
@@ -60,7 +70,88 @@ void registerBackendClassSpecificBindings(PyClass &) {} // pyclass
  * @param pyclass Nanobind's measurements class to bind methods.
  */
 template <class StateVectorT, class PyClass>
-void registerBackendSpecificMeasurements(PyClass &) {} // pyclass
+void registerBackendSpecificMeasurements(PyClass &pyclass) {
+    using PrecisionT =
+        typename StateVectorT::PrecisionT; // Statevector's precision
+    using ComplexT =
+        typename StateVectorT::ComplexT; // Statevector's complex type
+    using ParamT = PrecisionT;           // Parameter's data precision
```

Comment:
ParamT is a relic from the past. We can remove it in favor of using PrecisionT directly.